plutus-core/plutus-core/src/PlutusCore/Error.hs

```diff
@@ -86,7 +86,9 @@ data TypeError term uni fun ann
     = KindMismatch ann (Type TyName uni ()) (Kind ()) (Kind ())
     | TypeMismatch ann
         term
+        -- | Expected type
         (Type TyName uni ())
```

Comment:
I think that [breaks](https://github.com/haskell/haddock/issues/1065) haddock?

Reply:
Ok, going with the safer option :smile: